lib/synapse/service_watcher/zookeeper.rb

```diff
@@ -134,6 +134,28 @@ def create(path)
       @zk.create(path, ignore: :node_exists)
     end
 
+    # get with retry to reduce failure when network connectivity is flaky or ZK is overloaded
+    def get(path, opts = {})
```

Comment:
could we rename this function to `zk_get_path` or something more concrete?

Reply:
sure, thanks for the suggestion!